chromium_src/crypto/aead.h

```diff
@@ -0,0 +1,21 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CRYPTO_AEAD_H_
+#define BRAVE_CHROMIUM_SRC_CRYPTO_AEAD_H_
+
+// DO NOT use this function, this can only be called from
+// PasswordEncryptor::DecryptForImporter
+#define BRAVE_AEAD_H_                                                 \
```

Comment:
Maybe we can avoid patching the header if we redefine `NonceLength` for example and make `OverrideNonceLength` return size_t too? Something like:

<pre>
#define NonceLength \
  OverrideNonceLength(size_t length) { nonce_length_ = length; return length; } \
 \
 private: \
  size_t nonce_length_ = 0; \
 public: \
  size_t NonceLength
</pre>

Reply:
fixed in https://github.com/brave/brave-core/pull/10161/commits/2eb7d822d3de8e293d806f0fb3350aa724dd944a